packages/cosmos-client/index.ts

```diff
@@ -0,0 +1,13 @@
+import { CosmosClient } from "./src/CosmosClient";
+
+export interface ListVersionsResults {
+  [key: string]: string;
+}
+
+export interface PackageVersionsResponse {
+  results: ListVersionsResults;
+}
+
+module.exports = {
+  CosmosClient
+};
```

Comment:
Why not `export { CosmosClient };`?

Asking since this is the first occurrence of my editor tripping up on typings. With the above everything works fine.

Reply:
I’ll have to try that. I originally had 
export * from ‘./src/CosmosClient’; 

And it was failing at run-time. That’s why I moved the interfaces to this file. I’m not sure why it wasn’t working.

The module.exports is basically me defaulting back to something I knew would work.

Reply:
Sound totally reasonable!

I toyed around with the code a little and found [this](https://github.com/dcos/dcos-ui/pull/3587/commits/ee5bdeca52f5bab3ded69e25e523d9bad2c6b830) to be working - at runtime and for my editors type checking.